clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/jobs/local/ForceDestroyWatchdog.java

```diff
@@ -44,10 +46,17 @@ public synchronized void start(Process processToMonitor) {
   @Override
   public synchronized void timeoutOccured(final Watchdog w) {
     super.timeoutOccured(w);
+
+    try {
+      Thread.sleep(GRACE_PERIOD_MS);
+    } catch (InterruptedException e) {
+      // nothing to do
```

Comment:
Usually when ignoring an `InterrputedException` it's good to re-set the interrupt status on the thread via `Thread.currentThread().interrupt()` so that we don't lose the information that the thread was interrupted.  (This lets callers further up in the stack handle or ignore the exception, and prevents cases where we ignore interruption.)

I learned that [here](https://github.com/spinnaker/clouddriver/pull/3531#discussion_r271794353) when I also made an empty catch block in this case 😄 

In this case it probably doesn't make a big difference, but (as I learned on that linked PR) I think it's generally good practice.

Reply:
The more you know... I haven't seen that technique but totally makes sense